### PR TITLE
Configure enabled plugins from the shipit.yml and allow to pass a config hash

### PR DESCRIPTION
**Pull request by byroot**
I think it's relatively straightforward.

@gmalette @davidcornu fi you want to give it a quick look.

### CODE REVIEW

File: app/assets/javascripts/task/plugins.js.coffee [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@Plugins =
```
Comment (gmalette) [inline app/assets/javascripts/task/plugins.js.coffee]: I can't comment on the name, but I thought the preferred convention now was `file.coffee`, omitting the `.js`

Comment (gmalette) [inline app/assets/javascripts/task/plugins.js.coffee]: Should this be `Shipit.Plugins`?

Comment (byroot) [inline app/assets/javascripts/task/plugins.js.coffee]: Could be yes.

File: app/helpers/stacks_helper.rb [hunk 2]
```diff
@@ -66,7 +66,7 @@ def deploy_button_disabled?(commit)
   end
 
   def deploy_button_caption(commit)
-    state = commit.status
+    state = commit.significant_status.state
```
Comment (gmalette) [inline app/helpers/stacks_helper.rb]: no tests?

File: app/models/deploy_spec.rb [hunk 3]
```diff
@@ -109,6 +109,10 @@ def review_checks
     config('review', 'checks') || []
   end
 
+  def plugins
+    config('plugins') || {}
+  end
+
```
Comment (gmalette) [inline app/models/deploy_spec.rb]: tests?

File: app/helpers/shipit_helper.rb [hunk 4]
```diff
@@ -1,20 +1,16 @@
 module ShipitHelper
-  def include_plugins
-    tags = []
-    Rails.application.config.assets.paths.each do |path|
-      Dir[File.join(path, 'plugins/*')].each do |plugin_path|
-        tags << include_plugin_asset_tag(File.basename(plugin_path)) if File.file?(plugin_path)
-      end
-    end
-    tags.join.html_safe
+  def include_plugins(stack)
```
Comment (gmalette) [inline app/helpers/shipit_helper.rb]: TESSSTS!!

**Pull request merged**